truenas_install/__main__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding=utf-8 -*-
+import glob
+import json
+import logging
+import subprocess
+import sys
+
+logger = logging.getLogger(__name__)
+
+if __name__ == "__main__":
+    update = json.loads(sys.stdin.read())
+
+    root = update["root"]
```

Comment:
I wonder if we shouldn't create the new ZFS dataset from here so its unified between install and update?